src/shared/types/dash.ts

```diff
@@ -102,6 +102,10 @@ export interface DashData {
     supportDescription?: string;
 }
 
+export interface DashMeta {
+    description?: string;
```

Comment:
There is `is_release` field in dash meta.

https://github.com/datalens-tech/datalens-ui/blob/5dcf345f54807886e347a518467c4c4d52ef97b8/src/ui/units/dash/store/actions/base/actions.ts#L443